elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -1020,6 +1020,20 @@ public String getId(Object value) {
         return Collections.emptyList();
     }
 
+    /**
+     * Returns all annotations applied to an entity field.
+     *
+     * @param entityName  The entity name
+     * @param field  The entity field name
+     *
+     * @return a Collection of Annotations
+     *
+     * @throws NoSuchFieldException if the {@code field} is not defined in the entity
+     */
+    public Collection<Annotation> getFieldAnnotations(String entityName, String field) throws NoSuchFieldException {
+        return Arrays.asList(getEntityClass(entityName).getDeclaredField(field).getDeclaredAnnotations());
```

Comment:
I would make this consistent with other methods that do similar work (both in name and implementation):

```java
    public <A extends Annotation> A getAttributeOrRelationAnnotations(Class<?> entityClass,
                                         String identifier) {
        AccessibleObject fieldOrMethod = getEntityBinding(entityClass).fieldsToValues.get(identifier);
        if (fieldOrMethod == null) {
            return new ArrayList<>();
        }
        return fieldOrMethod.getDeclaredAnnotations();
    }
```